src/components/DropDownButton/index.tsx

```diff
@@ -108,11 +108,13 @@ const Toggle = ({
 			<JoinedButton
 				{...props}
 				icon={
-					open ? (
-						<FontAwesomeIcon icon={faChevronUp} />
-					) : (
-						<FontAwesomeIcon icon={faChevronDown} />
-					)
+					<Flex>
+						{props.icon}
+						<FontAwesomeIcon
+							style={{ marginLeft: props.icon ? '4px' : undefined }}
```

Comment:
@sradevski the alternative to this inline style could be to put an empty string between the two icons and update jest tests.